app/gui/src/project-view/components/ComponentDocumentation.vue

```diff
@@ -19,20 +19,19 @@ function docsForSelection() {
   return Ok(suggestionId)
 }
 
-const displayedId = computed(() =>
-  props.displayedSuggestionId != null ? Ok(props.displayedSuggestionId) : docsForSelection(),
-)
+const docs = computed(() => docsForSelection())
+// When the selection changes, we cancel the displayed suggestion override that can be in place.
+watch(docs, (_) => (displayedId.value = null))
+
+const displayedId = computed<SuggestionId | null>({
+  get: () => overrideDisplayed.value ?? unwrapOr(docs.value, null),
+  set: (value) => (overrideDisplayed.value = value),
+})
```

Comment:
This `computed` doesn't have the property that, after setting its value, the getter will return an equivalent value. That being the case, I don't see the benefit of a R/W computed here vs passing separate a separate getter and setter to `DocumentationPanel`.

Reply:
I’m not aware of such requirement on mutable computed, but you’re probably right, reverted it back to getter/setter pair.

Reply:
It's not a technical requirement, but I think it's a good convention to follow to avoid surprising behavior